extensions/wikia/UserLogin/js/FacebookFormConnectUser.js

```diff
@@ -0,0 +1,29 @@
+/* global UserBaseAjaxForm */
+
+(function () {
+	'use strict';
+
+	var FacebookFormConnectUser = function (el, options) {
+		UserBaseAjaxForm.call(this, el, options);
+	};
+
+	FacebookFormConnectUser.prototype = Object.create(UserBaseAjaxForm.prototype);
```

Comment:
Do we need to have a fallback if the browser doesn't support ```Object.create```? Or are we cutting those negative browsers out of our lives and just focusing on the future?